src/BinSkim.Driver/BinSkim.cs

```diff
@@ -10,6 +10,7 @@
 using System.Linq;
 using System.Runtime.InteropServices;
 using Microsoft.CodeAnalysis.Sarif;
+using Microsoft.CodeAnalysis.Sarif.Driver.Sdk;

```

Comment:
>CodeAnalysis.Sarif.Driver [](start = 16, length = 25)

is this change actually required? just curious

Reply:
Yeah, the namespace for EntryPointUtilities in the driver was changed at some point.